osdep/compiler.h

```diff
@@ -25,4 +27,17 @@
 #define MP_ASSERT_UNREACHABLE() (assert(!"unreachable"), abort())
 #endif
 
+#ifndef NDEBUG
+#define mp_assert assert
+#else
+#define mp_assert(expr)                                           \
+    do {                                                          \
+        if (!(expr)) {                                            \
+            fprintf(stderr, "Assertion failed: %s in %s:%d\n",    \
+                    #expr, __FILE__, __LINE__);                   \
+            abort();                                              \
+        }                                                         \
+    } while (0)
+#endif
```

Comment:
I don't really see the need for this.
`mp_raw_time_init` should do printf + abort if it can't find a clock source
the rest can just stay a normal assert

Reply:
Ok, if we don't want to have convenience macro for that.